crates/iroha_core/src/block.rs

```diff
@@ -160,7 +153,8 @@ mod pending {
                 .map(AsRef::as_ref)
                 .map(SignedTransaction::creation_time)
                 .max()
-                .expect("INTERNAL BUG: Block empty");
+                // Empty block is allowed
+                .unwrap_or(Duration::ZERO);
```

Comment:
```suggestion
                ;
```

this would be more semantically correct

Reply:
Do you suggest removing `unwrap_or` and making `latest_txn_time` of type `Option<Duration>`? Note that we already have the same pattern with `Duration::ZERO` for `prev_block_time`:

https://github.com/hyperledger-iroha/iroha/blob/d9683fcafbcf136f6305a3f98abd59294e8e84e9/crates/iroha_core/src/block.rs#L155-L156